monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js

```diff
@@ -5,7 +5,7 @@ import AuthComponent from '../../AuthComponent';
 import {faLaptopCode} from '@fortawesome/free-solid-svg-icons/faLaptopCode';
 import InlineSelection from '../../ui-components/inline-selection/InlineSelection';
 import {cloneDeep} from 'lodash';
-import {faCloud, faExpandArrowsAlt} from '@fortawesome/free-solid-svg-icons';
+import {faExpandArrowsAlt} from '@fortawesome/free-solid-svg-icons';
```

Comment:
Why is this change here? faCloud isn't used?

Reply:
Yep. Eslint got it and I thought why not.